lms/resources/_js_config/file_picker_config.py

```diff
@@ -0,0 +1,101 @@
+class FilePickerConfig:
+    """A config generator for specific file pickers."""
+
+    def __init__(self):
+        self._config_mapping = {
+            "blackboard": self.blackboard_config,
+            "canvas": self.canvas_config,
+            "google": self.google_files_config,
+            "vitalSource": self.vital_source_config,
+        }
```

Comment:
This can maybe move into `get_config()` below as that's the only place where it's used

Reply:
I actually moved this into the `__init__.py` stuff as it follows the pattern we saw in Course Copy. There's a testing gap in this version, as we don't mock ourselves. So instead of doing that, I've done it in the caller instead.